gtda/homology/grids.py

```diff
@@ -63,21 +74,24 @@ class CubicalPersistence(BaseEstimator, TransformerMixin):
     <http://gudhi.gforge.inria.fr/doc/latest/group__cubical__complex.html>`_.
```

Comment:
"Gudhi" should be "GUDHI" in hyperlink in Notes

Reply:
And "Cubical" should be lowercase there

Reply:
I can't comment on it as it is previous code, but I feel that the expression "filtrations of cubical complex" is not ideal or canonical? Might "filtered cubical complexes" be a good alternative?